sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionMessageEventArgs.cs

```diff
@@ -138,5 +149,16 @@ await _sessionReceiver.AbandonMessageAsync(message, propertiesToModify, cancella
             .ConfigureAwait(false);
             message.IsSettled = true;
         }
+
+        /// <summary>
+        /// Marks the session that is being processed as closed. No new receives will be initiated for the session before the
+        /// session is closed. Any already received messages will still be delivered to the user message handler, and in-flight message handlers
+        /// will be allowed to complete. Messages will still be completed automatically if <see cref="ServiceBusSessionProcessorOptions.AutoCompleteMessages"/>
+        /// is <value>true</value>.
```

Comment:
```suggestion
        /// is <c>true</c>.
```

`<value>` is intended to denote the value of a property and the docs tools would treat it as a string literal and render it verbatim if used in the summary.  _([ref](https://docs.microsoft.com/dotnet/csharp/programming-guide/xmldoc/value))_